charts/castai-db-optimizer/templates/deployment.yaml

```diff
@@ -24,6 +24,16 @@ spec:
         app.kubernetes.io/name: {{ include "name" . }}
     spec:
       terminationGracePeriodSeconds: 60
+      affinity:
```

Comment:
let's also add an option in helm chart values to disable host antiaffinity (enabled by default). might be needed for testing.